apparelo/install.py

```diff
@@ -36,10 +37,24 @@ def create_item_template():
     compacting.create_item_template()
     steaming.create_item_template()
     roll_printing.create_item_template()
+
 def create_attr_values():
     knitting.create_attr_values()
     knitting.create_additional_attribute()
+
 def remove_defaults():
     stock_setting=frappe.get_doc("Stock Settings")
     stock_setting.stock_uom=None
     stock_setting.save()
+
+def create_lot_warehouse():
```

Comment:
Do we also need to create a root supplier warehouse?

Reply:
Yes